Lint检查失败: scheduler.ts 中 async 方法缺少 await 表达式

## 测试输出错误信息

```
/Users/hs3180/Documents/disclaude/src/schedule/scheduler.ts
  368:3  error  Async method 'getCooldownStatus' has no 'await' expression  require-await
  384:3  error  Async method 'clearCooldown' has no 'await' expression      require-await
```

## 触发提交

CI Monitor 在检测到以下新提交后触发测试：
- d79ac35 fix(schedule): add missing await in cooldown methods (#945)
- 2b3ac60 fix(test): resolve type check errors in test files (Issue #941) (#943)

## 问题分析

提交 #945 声称修复了 cooldown 方法中缺少 await 的问题，但 Lint 检查仍然报告 `require-await` 错误。可能是：
1. 修复不完整，仍有方法被声明为 async 但没有使用 await
2. 或者需要移除不必要的 async 关键字

## 相关文件

- src/schedule/scheduler.ts (第 368 行和第 384 行)
- src/schedule/cooldown-manager.ts (可能相关)

## 测试环境

- 分支: main
- 本地 commit: d91a209 (更新前)
- 远程 commit: d79ac35 (更新后)

--- a/src/schedule/scheduler.ts
+++ b/src/schedule/scheduler.ts
@@ -372,7 +372,7 @@ ${task.prompt}`;
     remainingMs: number;
   } | null> {
     if (!this.cooldownManager) { return null; }
-    return this.cooldownManager.getCooldownStatus(taskId, cooldownPeriod);
+    return await this.cooldownManager.getCooldownStatus(taskId, cooldownPeriod);
   }
 
   /**
@@ -383,6 +383,6 @@ ${task.prompt}`;
    */
   async clearCooldown(taskId: string): Promise<boolean> {
     if (!this.cooldownManager) { return false; }
-    return this.cooldownManager.clearCooldown(taskId);
+    return await this.cooldownManager.clearCooldown(taskId);
   }
 }

